java/tests/com/google/privacy/differentialprivacy/CountTest.java

```diff
@@ -512,4 +512,156 @@ private static void testForBias(
+  }
+
+  @Test
+  public void computeConfidenceInterval_negativeLowerBound_clampsToZero() {
+    when(noise.computeConfidenceInterval(
+            anyLong(), anyInt(), anyLong(), anyDouble(), anyDouble(), anyDouble()))
+            .thenReturn(ConfidenceInterval.create(-5, 8));
+    count.computeResult();
+
+    // The result interval = (-5, 8), but count can't be negative, therefore it will be clamped to (0, 8).
+    assertThat(count.computeConfidenceInterval(0.152145599))
+            .isEqualTo(ConfidenceInterval.create(0, 8));
+  }
+
+  @Test
+  public void computeConfidenceInterval_clampsNegativeInfinityToZero() {
+    when(noise.computeConfidenceInterval(
+            anyLong(), anyInt(), anyLong(), anyDouble(), anyDouble(), anyDouble()))
+            .thenReturn(ConfidenceInterval.create(Double.NEGATIVE_INFINITY, 10));
+    count.computeResult();
+
+    // The result interval = (Double.NEGATIVE_INFINITY, 10), but count can't be negative, therefore it will be clamped to (0, 10).
+    assertThat(count.computeConfidenceInterval(0.152145599))
+            .isEqualTo(ConfidenceInterval.create(0, 10));
+  }
+
+  @Test
+  public void infiniteUpperBound_roundsToMaxLong() {
+    when(noise.computeConfidenceInterval(
+            anyLong(), anyInt(), anyLong(), anyDouble(), anyDouble(), anyDouble()))
+            .thenReturn(ConfidenceInterval.create(1, Double.POSITIVE_INFINITY));
+    count.computeResult();
+
+    // The result interval = (1, POSITIVE_INFINITY), but because of the long type, it will be (1, Long.MAX_VALUE).
+    assertThat(count.computeConfidenceInterval(0.152145599))
+            .isEqualTo(ConfidenceInterval.create(1, Long.MAX_VALUE));
+  }
+
+  @Test
+  public void computeConfidenceInterval_gaussianTest() {
+    // Mock the noise mechanism.
+    when(noise.computeConfidenceInterval(anyLong(), anyInt(), anyLong(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(invocation -> new GaussianNoise().computeConfidenceInterval(
+                    (Long) invocation.getArguments()[0],  (Integer) invocation.getArguments()[1], (Long) invocation.getArguments()[2],
+                    (Double) invocation.getArguments()[3],  (Double) invocation.getArguments()[4],  (Double) invocation.getArguments()[5]));
+    count =
+            Count.builder()
+                    .epsilon(0.5)
+                    .delta(0.9)
+                    .noise(noise)
+                    .maxPartitionsContributed(15)
+                    .build();
+    count.increment();
+    count.computeResult();
+
+    // The result interval = (-1, 3), but count can't be equal to 0, therefore it will be clamped to 0.
+    assertThat(count.computeConfidenceInterval(0.152145599)).isEqualTo(ConfidenceInterval.create(0,3));
+  }
+
+  @Test
+  public void computeConfidenceInterval_laplaceTest() {
+    // Mock the noise mechanism. Since noise is not Laplace, nor Gaussian, delta will be passed as a value instead of null, in order to pass the checks.
```

Comment:
this line (and several others) looks long, can you reformat this file?

Reply:
Yes, i reformatted it manually since the reformat tool doesn't shorten the comment lines.